src/clients/mdw-client.service.ts

```diff
@@ -0,0 +1,75 @@
+import { Injectable } from '@nestjs/common';
+import NETWORKS from '../lib/networks';
+import { nonNullable } from '../lib/utils';
+import {
+  AccountBalance,
+  BalancesV1,
+  ContractLog,
+  MdwPaginatedResponse,
+} from './mdw-client.model';
+
+@Injectable()
+export class MdwClientService {
+  constructor() {}
```

Comment:
do we need an empty constructor? can the class then be static? or is this a next thing?